### PR TITLE
Exercise 2: CSS Positioning and Box Model

### PR DESCRIPTION
**Pull request by Cathe313**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,6 +9,33 @@
...
+            Claritas est etiam processus dynamicus, qui sequitur mutationem
+            consuetudium lectorum. Mirum est notare quam littera gothica,
+            quam nunc putamus parum claram, anteposuerit litterarum formas
+            humanitatis per seacula quarta decima et quinta decima. Eodem
+            modo typi, qui nunc nobis videntur parum clari, fiant sollemnes
+            in futurum.
+        </p>
+    </div>
+    <button class="dialog-box__close"></button>
+</aside>
```
Comment (nyanofthemoon) [inline index.html]: @Cathe313 Please fix identation.

File: css/main.css [hunk 2]
```diff
@@ -1 +1,34 @@
...
+.dialog-box {
+    background-color: #d2d2d2;
+    width: 600px;
+    position: relative;
+    padding: 8px;
+}
+.dialog-box__title {
+    margin: 0px;
+    padding-bottom: 20px;
+    font-family: "Book Antiqua", serif;
```
Comment (nyanofthemoon) [inline css/main.css]: @Cathe313 :+1: [Book Antiqua is a "web-safe" font](http://www.cssfontstack.com/)

**Comment by Cathe313** [comment]
@nyanofthemoon Thanks - fixed!

**Comment by nyanofthemoon** [comment]
Perfect :+1:

**Pull request closed without merging**